ft/src/test/resources/projects/properties-override/gradle.properties

```diff
@@ -0,0 +1,6 @@
+license-management.version=1.0-SNAPSHOT
+
+# Placeholder values to be replaced by -Pmaven-license-parameters
+octopus-license-maven-plugin.version=0.0.0
```

Comment:
if you want to set wrong version you need to show this for us. Would you like to set version like 2.0.NOT_EXISTED?